modules/core/src/main/scala/com/azavea/stac4s/StacMediaType.scala

```diff
@@ -20,14 +20,15 @@ object StacMediaType {
     case "image/png"                                    => `image/png`
     case "image/jpeg"                                   => `image/jpeg`
     case "text/xml"                                     => `text/xml`
+    case "text/html"                                    => `text/html`
     case "application/xml"                              => `application/xml`
     case "application/json"                             => `application/json`
     case "text/plain"                                   => `text/plain`
     case "application/geo+json"                         => `application/geo+json`
     case "application/geopackage+sqlite3"               => `application/geopackage+sqlite3`
     case "application/x-hdf5"                           => `application/x-hdf5`
     case "application/x-hdf"                            => `application/x-hdf`
-    case s                                              => VendorMediaType(s)
+    case s                                              => VendorMediaType(s.split("vendor-").last)
```

Comment:
That made item to/from json roundtrip test fail.

Reply:
Also check this out: https://github.com/radiantearth/stac-spec/blob/2dc002400da3064a7ef4d2190fa8339d27aa2acf/item-spec/item-spec.md#media-types probably smth has changed

Reply:
Yeah, I'm not sure what the best option is here -- I don't think we should enforce a prefix (`vendor-` now, but probably should be `vnd.`) unless we are 100 percent sure we have all of them. Otherwise downstream applications end up in a tough spot (happens in Franklin already) if `stac4s` is missing a media type.

Reply:
@notthatbreezy also had the same thought

Reply:
We had this tracked as https://github.com/azavea/stac4s/issues/66 but hadn't gotten to it

Reply:
I think there are more of these where we do the vendor ser-/de-serialization wrong but they don't need to be part of your PR